internal/version/version_check_test.go

```diff
@@ -28,7 +28,7 @@ func TestCheck(t *testing.T) {
 		actual, err := s.Check(context.TODO())
 
 		assert.Empty(t, actual)
-		assert.NotEmpty(t, err)
+		assert.Error(t, err, "internal error")
```

Comment:
Just a note here that you are asserting that `err != nil` and putting `"internal error"` as the message to be displayed in case of `err == nil`.

Instead we could use `assert.ErrorContains(t, err, "internal error")`

Reply:
Updated the error assertion.